src/utils/formatAmount.ts

```diff
@@ -9,7 +9,7 @@ export { formatUnits }
 
 export const formatTokenAmount = (
   rawAmount: number | string,
-  format = "0,00.00",
+  format = "0,00",
```

Comment:
Are there any previous usages that need to be updated because they preferred `0,00.00`?

Reply:
I don't think so. I think we are ok with integers because I didn't find any numbers with decimals in Figma views.